app/src/main/java/com/example/playlistmaker/PlayerActivity.kt

```diff
@@ -12,10 +15,25 @@ import java.text.SimpleDateFormat
 import java.util.Locale
 
 class PlayerActivity : AppCompatActivity() {
+    companion object {
+        private const val STATE_DEFAULT = 0
+        private const val STATE_PREPARED = 1
+        private const val STATE_PLAYING = 2
+        private const val STATE_PAUSED = 3
+        private const val DELAY = 500L
+    }
+    private var playerState = STATE_DEFAULT
+    private lateinit var play: ImageView
+    private var mediaPlayer = MediaPlayer()
```

Comment:
```suggestion
    private val mediaPlayer = MediaPlayer()
```
Плеер не пересоздаётся в рамках этой активити, поэтому можно val использовать, он предпочтительнее